show_route/route/views.py

```
@@ -0,0 +1,308 @@
+
+def showmap(request):
+    return render(request,'showmap.html')
+
+import osmnx as ox
+import numpy as np
+import os
+import pandas as pd
+import geopandas as gd
+import networkx as nx
+import geopy
+from geopy.geocoders import Nominatim
```

Ruff (0.8.2)

`geopy.geocoders.Nominatim` imported but unused

Remove unused import: `geopy.geocoders.Nominatim`

(F401)